macros/staging/get_date_from_string.sql

```diff
@@ -0,0 +1,33 @@
+{% macro get_date_from_string(string_text) %}
+  {{ return(adapter.dispatch('get_date_from_string') (string_text)) }}
```

Comment:
```suggestion
  {{ return(adapter.dispatch('get_date_from_string', 'apple_store') (string_text)) }}
```
According these [docs](https://docs.getdbt.com/reference/dbt-jinja-functions/dispatch#for-package-maintainers) we gotta provide the namespace to `dispatch`